storage/aptosdb/src/lib.rs

```diff
@@ -1164,7 +1165,8 @@ impl DbReader for AptosDB {
 
     fn get_latest_state_checkpoint(&self) -> Result<Option<(Version, HashValue)>> {
         gauged_api("get_latest_state_checkpoint_version", || {
-            Ok(self.state_store.latest_checkpoint())
+            self.state_store
+                .get_state_snapshot_before(PRE_GENESIS_VERSION - 1)
```

Comment:
this is confusing, so PRE_GENESIS_VERSION - 1  == latest?

Reply:
yes

we are removing PRE_GENESIS shortly